prog/rand.go

```diff
@@ -119,10 +119,14 @@ func truncateToBitSize(v, bitSize uint64) uint64 {
 	return v & uint64(1<<bitSize-1)
 }
 
-func (r *randGen) randRangeInt(begin, end, bitSize uint64) uint64 {
+func (r *randGen) randRangeInt(begin, end, bitSize, align uint64) uint64 {
 	if r.oneOf(100) {
 		return r.randInt(bitSize)
 	}
+	if align != 0 {
```

Comment:
Hm. This block should be before the previous `r.oneOf(100)` block. Otherwise, we risk overflowing and we're doubling the chances of calling `randInt()`. I'll fix.

Reply:
But if we move it above, we will never generate unaligned value, right? I think we want unaligned once in a while too.

Reply:
Yep, you're right, and I was wrong regarding overflows, they'll still happen. I'm guessing overflows on generated values are not really a concern though (?)
Is doubling the chances of calling `randInt()` when we have an alignment parameter an issue? If not, we can leave it that way. If it is, I can rewrite to remove the recursive call.

Reply:
I would say it's fine. Probability of generating completely random values is quite low. I can also see some sense in generating completely random but aligned values (which we will get in case of recursion).

Reply:
Okay.